05_pytorch_backend/cnn_acceleration_tensorly_and_pytorch.ipynb

```diff
@@ -0,0 +1,594 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Accelerating deep neural networks with tensor decompositions\n",
+    "\n",
+    "In this notebook we will show how to use TensorLy for accelerating convolutional layers in PyTorch.\n",
+    "\n",
+    "This notebook is based on this blog post: https://jacobgil.github.io/deeplearning/tensor-decompositions-deep-learning\n",
+    "\n",
+    "We will use the Tucker and CP decomositions to make low rank approximations of convolutional layers, and make them:\n",
+    "1. Faster.\n",
+    "2. Smaller.\n",
+    "\n",
+    "Like pruning, these are tools that can be used to make the networks smaller and faster, and then try to restore accuracy by training further.\n",
+    "\n",
+    "---\n",
+    "\n",
+    "First lets get VGG19 from torch vision: "
```

Comment:
Let's load a pretrained VGG-19 network, using torch vision?
Also, it might be clearer to replace the `print(model)` by a short text explaining how the VGG net works (e.g. buch of conv, relu, pooling followed by two large FC layers before the final classification layer)